### PR TITLE
[Student][MBL-12476] Submission picker view

### PR DESCRIPTION
**Pull request by CalvinKern**
Refer to the AC on the jira ticket. This picker screen is just for file uploads. We will reuse this screen for confirming file/gallery selections on MEDIA SUBMISSIONS, but won't allow users to delete rows or add more items (only 1 attachment for media submissions).

File submissions are also hooked up to the submission service, so the picker is fully functional for those types of submissions.

### CODE REVIEW

File: apps/student/src/androidTest/java/com/instructure/student/ui/interaction/PickerSubmissionUploadInteractionTest.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ */
+package com.instructure.student.ui.interaction
+
+import com.instructure.canvas.espresso.Stub
+import com.instructure.student.ui.utils.StudentTest
+import org.junit.Test
+
+class PickerSubmissionUploadInteractionTest : StudentTest() {
+    override fun displaysPageObjects() = Unit
+
```
Comment (Blackchai) [inline apps/student/src/androidTest/java/com/instructure/student/ui/interaction/PickerSubmissionUploadInteractionTest.kt]: meta annotations please!

File: apps/student/src/androidTest/java/com/instructure/student/ui/renderTests/PickerSubmissionUploadRenderTest.kt [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class PickerSubmissionUploadRenderTest : StudentRenderTest() {
+
+    private val page = PickerSubmissionUploadRenderPage()
+    private val baseVisibilities = PickerVisibilities(fab = true)
+
+    @Test
```
Comment (Blackchai) [inline apps/student/src/androidTest/java/com/instructure/student/ui/renderTests/PickerSubmissionUploadRenderTest.kt]: meta annotations!

Comment (CalvinKern) [reply apps/student/src/androidTest/java/com/instructure/student/ui/renderTests/PickerSubmissionUploadRenderTest.kt]: lol, oops. Didn't realize those got merged in. Will definitely add those!

Comment (Blackchai) [reply apps/student/src/androidTest/java/com/instructure/student/ui/renderTests/PickerSubmissionUploadRenderTest.kt]: :D

File: apps/student/src/test/java/com/instructure/student/test/assignment/details/submission/PickerSubmissionUploadEffectHandlerTest.kt [hunk 3]
```diff
@@ -298,4 +375,137 @@ class PickerSubmissionUploadEffectHandlerTest : Assert() {
...
+        verify(exactly = 0) {
+            eventConsumer.accept(any())
+        }
+
+        confirmVerified(eventConsumer)
+    }
+
+    @Test
+    fun `isPickerRequest with valid codes return true`() {
+        assertTrue(
```
Comment (Blackchai) [inline apps/student/src/test/java/com/instructure/student/test/assignment/details/submission/PickerSubmissionUploadEffectHandlerTest.kt]: Can we break these into multiple tests?

**Pull request merged**